src/components/structures/RoomSubList.js

```diff
@@ -37,6 +37,42 @@ import toRem from "../../utils/rem";
 // turn this on for drop & drag console debugging galore
 const debug = false;
 
+class RoomTileErrorBoundary extends React.PureComponent {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            error: null,
+        };
+    }
+
+    static getDerivedStateFromError(error) {
+        // Side effects are not permitted here, so we only update the state so
+        // that the next render shows an error message.
+        return { error };
+    }
+
+    componentDidCatch(error, { componentStack }) {
+        // Browser consoles are better at formatting output when native errors are passed
+        // in their own `console.error` invocation.
+        console.error(error);
+        console.error(
+            "The above error occured while React was rendering the following components:",
+            componentStack,
+        );
+    }
+
+    render() {
+        if (this.state.error) {
+            return (<div className="mx_RoomTile mx_RoomTileError">
```

Comment:
Is `mx_RoomTileError` defined somewhere? Perhaps also post a screenshot of what the error tile looks like if you can.

Reply:
it's not and it looks like ass, just a room ID so that it's not a big empty space

Reply:
not a big deal but why merge it anyway?